packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/medications-summary.component.tsx

```diff
@@ -13,7 +13,29 @@ interface MedicationSummaryProps {
 const MedicationSummary: React.FC<MedicationSummaryProps> = ({ medications }) => {
   const { t } = useTranslation();
 
-  const drugOrders = medications?.filter((medication) => medication?.order?.orderType?.display === 'Drug Order');
+  const drugOrders = medications?.filter((medication) => {
+    // First, check if it's a drug order
+    if (medication?.order?.orderType?.display !== 'Drug Order') {
+      return false;
+    }
+
+    // Then filter out discontinued orders
+    if (medication?.order?.action === 'DISCONTINUE') {
```

Comment:
Instead of filtering these out, should we potentially show these with a tag that says `discontinued`, `stopped` or `expired`?

Reply:
Let's handle this nuance as a follow-up enhancement.